lua/entities/gmod_wire_expression2/core/custom/cfc_vector2_NumberOperands.lua

```diff
@@ -0,0 +1,396 @@
+E2Lib.RegisterExtension( "cfc_e2_lib", true )
```

Comment:
This file needs to be its own PR. I won't review it in here.

Reply:
This file also needs to be converted to spaces.

Reply:
What do you mean by converted to spaces?

Reply:
the file is indented with tabs, files should be indented with 4 spaces

Reply:
thank you